### DISCUSSION TITLE
snap update stuck?

### DISCUSSION THREAD

**Discussion by MichelHanzen** [zwave-js/zwave-js-ui#2864] [Ideas]
Here is the result of the command: snap info zwave-js-ui
.
.
snap-id: bKAUrYAOTfQjguWrb9V3CrABqC01QIhC
channels:
  latest/stable:    v8.4.1 2022-11-19 (43) 151MB -
  latest/candidate: v8.4.1 2022-11-17 (43) 151MB -
  latest/beta:      ↑                            
  latest/edge:      v8.5.0 2022-11-19 (45) 149MB -

The latest version available is 8.6.2 released on December 23. Is there a problem with the distribution of the software through snap?

**Comment by robertsLando** [comment]
cc @jmgiaever

**Comment by jmgiaever** [comment]
Yeah, I haven't had a minute to sit down behind the computer yet since it was released. And what I understood there's no critical fixes shit this release?

**Comment by MichelHanzen** [comment]
I would just like to be sure that my move from zwavejs2mqtt to zwavejsui was done succesfully.
Happy New Year!

**Comment by jmgiaever** [comment]
It's published now. You can close this @robertsLando

**Reply by robertsLando** [reply]
It's a discussion so I cannot close but I can mark as answered :)

**Reply by jmgiaever** [reply]
Hah, okei. Didn't scroll to the top. 🤪
